.config/foot/foot.ini

```
@@ -0,0 +1,2 @@
+[colors]
+ alpha=0.4
```

```suggestion
 alpha=0.7
```

actually that window opacity should be `window.opacity: 0.4` and nobody complained so far, I fixed it locally and forgot to xi it here.

0.4 looks a little bit too much on bright background. How 0.4 works for you ?

---

yep, 0.8 probably works better